src/OpenLoco/src/Windows/CompanyFaceSelection.cpp

```diff
@@ -29,7 +29,9 @@ using namespace OpenLoco::Interop;
 namespace OpenLoco::Ui::Windows::CompanyFaceSelection
 {
     static loco_global<CompanyId, 0x9C68F2> _9C68F2; // Use in a game command??
-    static loco_global<uint16_t, 0x112C1C1> _numberCompetitorObjects;
+
+    // Count was previously 0x112C1C1
+    static std::vector<std::pair<ObjectManager::ObjectIndexId, ObjectManager::ObjectIndexEntry>> _competitorList;
```

Comment:
I think this could do with some type aliasing tbh.

Reply:
The neat thing is there are already two